src/test/java/io/sudhakar/student/controller/StudentControllerTest.java

```diff
@@ -386,7 +386,7 @@ void updateStudentInternalServerError() {
         when(studentService.updateStudent(1, student))
                 .thenReturn(serviceResponse);
 
-        ResponseEntity<?> responseEntity = studentController.updateStudent(1, student);
+        ResponseEntity<Void> responseEntity = studentController.updateStudent(1, student);
         assertEquals(500, responseEntity.getStatusCodeValue());
 
     }
```

Comment:
ResponseEntity<Void> change in comments something like this

Reply:
comments updated